asm-to-pil/src/vm_to_constrained.rs

```diff
@@ -355,17 +355,19 @@ impl<T: FieldElement> VMConverter<T> {
         let inputs: Vec<_> = params
             .inputs
             .into_iter()
-            .map(|param| match param.ty {
-                Some(ty) if ty == "label" => Input::Literal(param.name, LiteralKind::Label),
-                Some(ty) if ty == "signed" => {
-                    Input::Literal(param.name, LiteralKind::SignedConstant)
-                }
-                Some(ty) if ty == "unsigned" => {
-                    Input::Literal(param.name, LiteralKind::UnsignedConstant)
-                }
-                None => Input::Register(param.name),
-                Some(ty) => panic!("Invalid param type {}", ty),
-            })
+            .map(
+                |param| match param.ty.as_ref().and_then(|ty| ty.try_to_identifier()) {
+                    Some(ty) if ty == "label" => Input::Literal(param.name, LiteralKind::Label),
+                    Some(ty) if ty == "signed" => {
+                        Input::Literal(param.name, LiteralKind::SignedConstant)
+                    }
+                    Some(ty) if ty == "unsigned" => {
+                        Input::Literal(param.name, LiteralKind::UnsignedConstant)
+                    }
+                    None => Input::Register(param.name),
```

Comment:
I think we should panic if the type is not an identifer. Currently, this is just ignored.